generators/app/templates/javascript/server.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
```

Comment:
voxa3 has a builtin server, unless we want multiple platforms, let's just use that